README.md

```diff
@@ -62,21 +62,11 @@ operators][operators].
 
 ## Projects using operatorkit
 
-- https://github.com/giantswarm/app-operator
-- https://github.com/giantswarm/aws-operator
-- https://github.com/giantswarm/azure-operator
-- https://github.com/giantswarm/cert-operator
-- https://github.com/giantswarm/cluster-operator
-- https://github.com/giantswarm/chart-operator
-- https://github.com/giantswarm/flannel-operator
-- https://github.com/giantswarm/kvm-operator
-- https://github.com/giantswarm/node-operator
-- https://github.com/giantswarm/prometheus-config-controller
-- https://github.com/giantswarm/release-operator
-- more to come
+[Giant Swarm operators using operatorkit](https://github.com/search?p=1&q=topic%3Aoperator+org%3Agiantswarm&type=Repositories).
 
 ## Example
```

Comment:
Something is weird with the `Example` heading. 

<img width="415" alt="Screenshot 2019-11-11 at 13 24 07" src="https://user-images.githubusercontent.com/552769/68587120-9333dd80-0486-11ea-801d-bb0693d8efe9.png">

Reply:
Thanks and it was because the search term is URL encoded. Using a citation style link fixed it.